### DISCUSSION TITLE
[Proposal]beatmap label clicking

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#12999] [Ideas]
when there's only one difficulty in beatmapset if you click main label not difficulty label it could start gameplay too

**Comment by Gabixel** [comment]
~Can be interesting for songs with multiple difficulties too, like a substitute for the osu! logo on bottom-right corner to play the selected difficulty. Not really useful but seems more natural I guess~ (Actually I changed my mind, I agree with peppy's choice)

**Comment by peppy** [comment]
I disagree this feels more natural. When clicking the easer it should probably flash the difficulties to let the player know where to click, though.
